programming/sr/vision/markers.md

```diff
@@ -21,13 +21,13 @@ Details of the types and size of markers used in the game can be found in the [r
 You can download all the markers as a single [ZIP file](/docs/resources/2023/sr-markers-sr2023.zip) or individually from the [git repo](https://github.com/srobo/game-markers/tree/master/SR2023/markers).
 
 You must ensure that your PDF viewer is not resizing the documents when printing.
-This can be checked by measuring the grey box around the marker and comparing this to the size defined in the rules.
+This can be checked by measuring the black marker and comparing this to the size defined in the rules.
```

Comment:
For my understanding: do AprilTags markers always have at least one square's worth of black all the way around the outside of the marker?

Reply:
Yes, that's correct. We then add an extra bit of padding (coincidentally 1 square) to help contrast between the backing surface and marker.